flink-connectors/flink-jdbc/src/test/java/org/apache/flink/api/java/io/jdbc/JDBCInputFormatTest.java

```diff
@@ -154,6 +154,63 @@ public void testFetchSizeCanBeConfigured() throws SQLException {
 		Assert.assertEquals(desiredFetchSize, jdbcInputFormat.getStatement().getFetchSize());
 	}
 
+	@Test
+	public void testAutoCommitFalse() {
```

Comment:
I am not sure this is needed, unlike setFetchSize, there's no Precondition.check in setAutoCommit, maybe just combine these with the last test?